conda/core/solve.py

```diff
@@ -1348,6 +1348,10 @@ def _configure_solver_for_remove(self, state):
             list(set(chain(self._history_specs(), aggresive_updates))),
             api.SOLVER_USERINSTALLED,
         )
+        not_installed = [s.name for s in self.specs_to_remove if s.name not in installed_names]
```

Comment:
Let's leave a comment about the ticket here and use 
```suggestion
        not_installed = [spec.name for spec in self.specs_to_remove if spec.name not in installed_names]
```